File: 1051 Height Checker/1051height-checker.ts
function heightChecker(heights: number[]): number {
    const expected: Array<number> = heights.toSorted((a: number, b: number) => a - b)
    
    let count: number = 0
    
    for(let i: number = 0; i < heights.length; i++){
        if(expected[i] !== heights[i]){
            count++
        }
    }
    
    return count
};

File: 1051 Height Checker/1051height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int n = heights.length;
        int[] expected = Arrays.copyOf(heights, n);
        
        Arrays.sort(expected);
        
        int output = 0;
        
        for(int i = 0; i < n; i++){
            if(expected[i] != heights[i]){
                output++;
            }
        }
        
        return output;
    }
}